### ISSUE TITLE
[UI Bug] Accessibility Menu overlaps with youtube icon

### ISSUE THREAD

**Issue by sundeshgupta** [topcoder-platform/Training-Challenges#126]
**Steps to Reproduce:**
1. Go to: [www.topcoder.com](url)
2. Scroll to the bottom of the page.

**Actual Result:**
Accessibility icon overlaps with youtube icon at the bottom right of the page. Even after clicking on 'hide' (which hides accessibility icon), the click-box of youtube icon is very small, making it difficult to click on the youtube link. 

**Expected Result:** 
Accessibility icon should have a minimum bottom margin.

**Screenshots or screencast URL:**
![Screenshot from 2020-05-17 21-02-44](https://user-images.githubusercontent.com/43722047/82153236-e21e5d80-9883-11ea-8e0d-3a0488e0608a.png)
![Screenshot from 2020-05-17 21-03-20](https://user-images.githubusercontent.com/43722047/82153245-eea2b600-9883-11ea-8266-cf4c28cb2710.png)

**Device/OS/Browser Information:**
OS: Ubuntu 18.04.4 LTS
Browser: Google Chrome 80.0.3987.162

**Comment by nithyaasworld** [comment]
Good Job @sundeshgupta
